### ISSUE TITLE
Issues with DUE

### ISSUE THREAD

**Issue by Colatino** [tomstewart89/Geometry#8]
Hi!

When compiling for the DUE I get the following error:

> In file included from e:\Andre\Downloads\conversion\conversion.ino:1:0:
> E:\Andre\Documentos\Arduino\libraries\Geometry/Geometry.h:12:22: error: reference to 'Matrix' is ambiguous
>  class Point : public Matrix<3,1>

I changed the library files so that all Matrix occurences had a prepending BLA:: (like advised on BasicLinearAlgebra git) and the compilation error vanished.

Still didn't test to see if the calculations are correct with this change.

**Comment by tomstewart89** [comment]
Hey @Colatino,

Sorry about the incredibly slow reply. I've been giving this library some well needed attention over the last few weeks and I'm pretty sure this will have fixed this issue. If you have any more trouble with it or anything else in v2.0, let me know!

**Comment by pegasusbel** [comment]
Sorry to get back on this topic. Got similar error with Geometry 2.0.0 and BasicLinearAlgebra 3.3.0. Haven't yet modded any files from the lib's.

Using IDE 1.8.15, SAM boards 32-bit ARM Cortex-M3 version 1.6.12.

Just made a blank sketch to illustrate:
`#include <BasicLinearAlgebra.h>
#include <Geometry.h>
using namespace Geometry;
using namespace BLA;

void setup() {
  // put your setup code here, to run once:

}

void loop() {
  // put your main code here, to run repeatedly:

}`

And the compiler reports as in the text file...
[errorMessages.txt](https://github.com/tomstewart89/Geometry/files/8019481/errorMessages.txt)

**Comment by tomstewart89** [comment]
Hey @pegasusbel , thanks for the info. It looks like those boards define a `Matrix` class which conflicts with the one I defined over in BasicLinearAlgebra. I just made a fix and put up a new release (v2.1) which should resolve the problem, if you have any more trouble feel free to let me know!

I should also mention, because of this conflict, it's probably better to avoid `using namespace BLA` and to declare any matrices you might need like so: `BLA::Matrix<N,M> mat;`
